pages/lists/[listId]/edit.tsx

```diff
@@ -207,133 +216,203 @@ export default function EditListPage({ list, initialContributors }: EditListPage
+                        <Pagination
+                          pages={new Array(meta.pageCount).fill(0).map((_, index) => index + 1)}
+                          pageSize={5}
+                          hasNextPage={meta.hasNextPage}
+                          hasPreviousPage={meta.hasPreviousPage}
+                          totalPage={meta.pageCount}
+                          page={meta.page}
+                          onPageChange={function (page: number): void {
+                            setPage(page);
+                          }}
+                          showTotalPages={false}
+                        />
+                      </div>
+                    </>
+                  ) : (
+                    <p className="text-light-slate-11">No contributors to remove found.</p>
+                  )}
+                </>
+              )}
+            </>
           </div>
-          <div className="flex xs:items-center xs:justify-between xs:flex-row flex-wrap pb-8 border-b border-solid border-light-slate-6 flex-col justify-start">
-            <h2 className="text-light-slate-12 ">Add Contributors</h2>
-            <Button
-              variant="outline"
-              href={`/lists/${list.id}/add-contributors`}
-              className="flex gap-2.5 items-center cursor-pointer w-min mt-2 sm:mt-0"
-            >
-              <FaUserPlus size={22} />
-              <span>Add new contributors</span>
-            </Button>
-          </div>
-        </form>
-        <div className="flex flex-col pb-4 gap-4">
-          <h2 className="text-light-slate-12">Remove Contributors</h2>
-          <div className="flex flex-col w-full gap-2 md:flex-row">
-            <label className="flex w-full flex-col gap-4">
-              <span className="sr-only">Search your contributor list</span>
-              <Search
-                placeholder="Search your contributor list"
-                className="!w-full text-sm py-1.5"
-                name={"contributors"}
-                onChange={(value) => setContributorSearchTerm(value)}
-              />
-            </label>
+          <div className="flex flex-col gap-4 py-6 border-t border-b border-light-slate-8">
+            <label className="text-light-slate-12">Danger Zone</label>
+            <div className="flex flex-col p-6 rounded-2xl bg-light-slate-4">
+              <Title className="!text-1xl !leading-none !border-light-slate-8 border-b pb-4" level={4}>
+                Delete List
+              </Title>
+              <Text className="my-4">Once you delete a list, you&#39;re past the point of no return.</Text>
+
+              <div>
+                <Button
+                  onClick={() => handleOnDelete()}
+                  variant="default"
+                  className="border bg-light-red-6 border-light-red-8 hover:bg-light-red-7 text-light-red-10"
+                >
+                  Delete list
+                </Button>
```

Comment:
refactor (blocking): use the button's `destructive` variant.

```suggestion
                <Button onClick={() => handleOnDelete()} variant="destructive">
                  Delete list
                </Button>
```

Note: Not for this pull request, but the same will need to be done for the Delete Page button in the edit insights page. I'll create an issue for this.

Reply:
#2434 has been created for the Delete page button on the edit insights page.